src/GitVersion.Core.Tests/VersionCalculation/VersionSourceTests.cs

```diff
@@ -27,8 +28,8 @@ public void VersionSourceSha()
         var nextVersion = nextVersionCalculator.FindVersion();
 
         nextVersion.IncrementedVersion.BuildMetaData.ShouldNotBeNull();
-        nextVersion.IncrementedVersion.BuildMetaData.VersionSourceSha.ShouldBe(initialCommit.Sha);
-        nextVersion.IncrementedVersion.BuildMetaData.CommitsSinceVersionSource.ShouldBe(2);
+        nextVersion.IncrementedVersion.BuildMetaData.VersionSourceSha.ShouldBeNull();
```

Comment:
That's a breaking change yes. We need to discuss it or I need to explain this point from my point of view:

VersionSourceSha normally gives you the commit which is dependent for the given SematicVersion. Here it is the next version 0.1.0 (Increment.Patch). Here it comes from the fallback strategy which has per definition no commit which is responsable for it. It makes or I'm wrong?

Reply:
I see. Whether `VersionSource` for `FallbackVersionStrategy` should be `null` or the initial commit can be convincingly argued either way, imho. If `null` makes a positive impact on our codebase, performance or similar, then I'm all for it, but if it's just a semantic or preferential change, I think we should keep it as is.

Reply:
I think it has a influence on the CommitsSinceVersionSource. That's the initial reason why I have changed it. Because in my opinion we have not two we have three commits in this sceanrio

Reply:
That's is also the reason why so many tests are now given +1 on the pre-release version number

Reply:
Done